cmd/janitor.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ory/x/configx"
+)
+
+var janitorCmd = &cobra.Command{
+	Use:   "janitor <database-url>",
+	Short: "Clean the database of old tokens and login/consent requests",
+	Long: `This command will cleanup any expired oauth2 tokens as well as login/consent requests.
+
+	### Warning ###
+
+	This is a destructive command and will purge data directly from the database.
+	Please use this command with caution if you need to keep historic data for any reason.`,
+	Run: cmdHandler.Janitor.Purge,
+}
+
+func init() {
+	RootCmd.AddCommand(janitorCmd)
+	janitorCmd.Flags().StringP("keep-if-younger", "k", "", "Keep database records that are younger than a specified duration e.g. 1s, 1m, 1h.")
```

Comment:
Do you mean this in light of the `consent-request-lifespan`, `refresh-lifespan` and `access-lifespan` flags?

Reply:
yes, all of them!

Reply:
The `consent-request-lifespan` will clash with config "c" flag so will have to make it shorthand `o`.

Shouldn't I rather make this a slice of string rather and get rid of all the flags? So e.g. `janitorCmd.Flags().StringSliceP("keep-if-younger", "k", []string{}, "Keep database records that are younger than a specified duration of options [notAfter, access-lifespan, refresh-lifespan, consent-request-lifespan] e.g. 1s, 1m, 1h, 2h.")`

This will result in k[0] =1s, k[1]=1m, k[2]=1h and k[3]=2h. Might be a bit cleaner in terms of flags?

Reply:
No, this is a very confusing API to use where it's easy to make typos!

The `-o` flag is also reserved for output and should not have other meanings, similar as `-f` means force. Maybe it's best to not use shorthand flags here at all and instead require the long arguments!